index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Emmanuel and Tressor</title>
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
- I think it would be better to put your _<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>_  at the bottom of your body section